cdap-ui/app/features/services/controllers/tabs/runs/tabs/log-ctrl.js

```diff
@@ -21,6 +21,10 @@ angular.module(PKG.name + '.feature.services')
       return;
     }
 
+    this.getDownloadFileName = () => {
```

Comment:
CDAP apps follow the same naming convention. Hydrator and Tracker will also receive their custom naming convention. @ajainarayanan will cover Hydrator in his PR, and I will cover Tracker once it has been integrated.